### PR TITLE
slotPropGetter - Allow to style cells

### PR DESCRIPTION
**Pull request by bkniffler**
Allows for something like:

ohne titel

### CODE REVIEW

**Comment by bkniffler** [comment]
Hey, any news on this?

File: src/Calendar.js [hunk 1]
```diff
@@ -242,6 +242,18 @@ class Calendar extends React.Component {
    eventPropGetter: PropTypes.func,
 
    /**
+    * Optionally provide a function that returns an object of className or style props
+    * to be applied to the the time-slot node.
```
Comment (jquense) [inline src/Calendar.js]: can we add a note here advising caution? styles that change layout or position may break the calendar in unexpected ways.

**Comment by jquense** [comment]
sorry slipped past me. looks good just one comment

**Comment by bkniffler** [comment]
Okidoki!

**Comment by jquense** [comment]
thanks!

**Pull request merged**